_ingest-pipelines/processors/ml-inference.md

```diff
@@ -108,14 +113,37 @@ PUT /_ingest/pipeline/ml_inference_pipeline
 ```
 {% include copy-curl.html %}
 
+For externally hosted models, in a Predict API request, all fields are usually nested inside the `parameters` object:
```

Comment:
Usually, or will always be? From my understanding, it is always.

Reply:
Since the ml inference processor makes this assumption within the mapping / conversion logic, I think we should make this a bit clearer.

Reply:
cc @mingshl

Reply:
We are specifying here to the user that they need not give parameters path [here](https://github.com/opensearch-project/documentation-website/pull/7368/files#diff-b777c14a01b3fe5457f898d2f6a8d550f68c76ef49629d5fd436917c9c4080f8R131). Should we add more info?